supero-front/src/Activity.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import {
+  faBolt,
+  faClock,
+  faMapMarkerAlt,
+  faUser
+} from "@fortawesome/free-solid-svg-icons";
+library.add(faBolt, faClock, faMapMarkerAlt, faUser);
+
+const sports = ["Running", "Vélo", "Football", "Sport Ext.", "Sport Int."];
+
+const Activity = props => {
```

Comment:
New best practice to apply https://reactjs.org/docs/typechecking-with-proptypes.html